### PR TITLE
User Management Test cases

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
Jira: https://mattermost.atlassian.net/browse/MM-56151
Dev PR: https://github.com/mattermost/mattermost/pull/26055

### CODE REVIEW

**Review by devinbinnie — APPROVED** [review]

**Review by marianunez — COMMENTED** [review]
@yasserfaraazkhan can we include tests for the case of mysql not showing the agregated data points? https://mattermost.atlassian.net/browse/MM-56615 

@devinbinnie can you help on what to expect in that case? I can't find it in the designs

File: data/test-by-folder.json [hunk 1]
```diff
@@ -6087,6 +6087,15 @@
       "When deactivating users in the System Console, email address should not disappear"
     ]
   },
+  {
+    "folder": "suite/system-console/user-management/new-ui",
+    "tests": [
+      "Ability to Export Data after upgrading to Professional license",
+      "Ability to Sort the columns and view only desired columns",
+      "Ability to search by test and apply filters",
+      "Ability to view User Profiles in User management section"
```
Comment (marianunez) [inline data/test-by-folder.json:6096]: Also, can we cover the case of data range filters?

File: data/test-cases/suite/system-console/user-management/new-ui/test-1.1-view.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Test requirements:
+
+1. Have an instance with server version >= 9.6.
+
+Steps:
+
+1. Login into mattermost with admin account.
+2. Navigate to System Console > Users.
+3. In the first column, click on the user name.
+4. Verify the User Configuration page is opened.
```
Comment (marianunez) [inline data/test-cases/suite/system-console/user-management/new-ui/test-1.1-view.md:47]: Do we want to also include to check the availability of the user actions menus ?

File: data/test-cases/suite/system-console/user-management/new-ui/test-1.3-export.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+6. Verify that the Export functionality is now available.
+
+Scenario: Export Data for different durations
+
+1. Login into mattermost with admin account.
+2. Navigate to System Console > Users.
+3. Under `Duration` on the top right side, select `Last 30 days`.
+4. Click on `Export` button
+5. Verify the Admin receives a message from system-bot stating `You've started an export of user data for the last 30 days. When the export is complete, a CSV file will be delivered to you in this direct message.`
+
```
Comment (marianunez) [inline data/test-cases/suite/system-console/user-management/new-ui/test-1.3-export.md:81]: Can we add another step here to validate the actual data received in the CSV?

**Pull request merged**